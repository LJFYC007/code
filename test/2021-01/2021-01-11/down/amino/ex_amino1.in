4 8
Q ****
I 0101 5
I 1110 3
Q ****
Q *0**
Q ***0
I 0101 -3
Q *1**
