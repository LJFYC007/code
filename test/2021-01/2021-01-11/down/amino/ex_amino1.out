0
8
0
3
5
